cmd/dlv/cmds/commands.go

```diff
@@ -188,6 +193,9 @@ While --continue is not supported, stopOnEntry launch/attach attribute can be us
 execution is resumed at the start of the debug session.`,
 		Run: dapCmd,
 	}
+	dapCommand.Flags().StringVar(&dapConnect, "connect", "", "host:port of the proxy DAP server. If set, the command starts a DAP server in reverse mode")
+	dapCommand.Flags().MarkHidden("connect")
```

Comment:
Personally I wouldn't hide it, but either way is fine. The documentation mentioning a proxy however is confusing, because the other end of the connection doesn't have to be a proxy. I think it should say: `when this option is used Delve connects to a DAP client listening on host:port, instead of listening for connections`.